tests/e2e/environment-workflow.test.ts

```diff
@@ -1,374 +1,69 @@
+import { describe, test, expect, beforeAll } from 'vitest';
 import {
-  describe,
-  test,
-  expect,
-  beforeAll,
-  afterAll,
-  afterEach,
-  vi
-} from 'vitest';
-import { getTestWorkerUrl, WranglerDevRunner } from './helpers/wrangler-runner';
-import {
-  createSandboxId,
-  createTestHeaders,
-  cleanupSandbox
-} from './helpers/test-fixtures';
-import type {
-  EnvSetResult,
-  ExecResult,
-  WriteFileResult,
-  Process,
-  ProcessLogsResult
-} from '@repo/shared';
-
-describe('Environment Variables Workflow', () => {
-  describe('local', () => {
-    let runner: WranglerDevRunner | null;
-    let workerUrl: string;
-    let currentSandboxId: string | null = null;
-
-    beforeAll(async () => {
-      // Get test worker URL (CI: uses deployed URL, Local: spawns wrangler dev)
-      const result = await getTestWorkerUrl();
-      workerUrl = result.url;
-      runner = result.runner;
+  getSharedSandbox,
+  createUniqueSession
+} from './helpers/global-sandbox';
+import type { ExecResult } from '@repo/shared';
+
+/**
```

Comment:
can/should we expand this test to cover all the ways in which it's possible to set envs and whether they're getting overridden properly?
1. Via ENV in dockerfile
2. Via setEnvVars at the sandbox level
3. Via setEnvVars at the session level
4. Passing `env` to exec/execStream/startProcess